lib/main_common.pm

```diff
@@ -265,6 +267,10 @@ sub is_kernel_test {
         || get_var('TUNED'));
 }
 
+sub is_container_test {
+    return (check_var('EXTRATEST', 'cont'));
+}
+
```

Comment:
Yes, it is used in sle/main.pm

Reply:
Then let's put it at least to `main_containers.pm`.

Reply:
But it will fail `main_containers.pm` is not imported in main.pm
Mostly all the `is_X_test` functions are declared here: 
```
.//main_pods.pm:52:sub is_pod_test {
.//main_common.pm:253:sub is_ltp_test {
.//main_common.pm:258:sub is_kernel_test {
.//main_common.pm:270:sub is_container_test {
.//main_common.pm:312:sub is_updates_tests {
.//main_common.pm:318:sub is_updates_test_repo {
.//main_common.pm:333:sub is_memtest {
```
Would it make sense to move it to `containers/utils.pm`? If so, then I need to import it in `sle/main.pm` anyway. If we keep it here, we avoid extra imports.

Reply:
I'd also move it to `containers/utils.pm` similar to `is_publiccloud` (https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/publiccloud/utils.pm#L56)

Reply:
done